docs/creating_reading.rst

```diff
@@ -35,6 +35,60 @@ accessing other Stokes parameters, see :doc:`stokes`.
           files. See the :doc:`big_data` page for more details about dealing
           with large data sets.
 
+
+Reading images from file
+------------------------
+
+While spectral-cube is designed for cube analysis, in the course of normal
+analysis procedures you are likely to need to load up one- and two-dimensional
+subsets or views of the data.
+
+You can load :class:`~spectral_cube.lower_dimensional_objects.Projection`
+objects from FITS HDU objects with
+:meth:`~spectral_cube.lower_dimensional_objects.Projection.from_hdu`.  Only
+FITS reading is supported::
```

Comment:
```suggestion
FITS reading is currently supported::
```
Raised a separate issue for loading 2D CASA images: https://github.com/radio-astro-tools/casa-formats-io/issues/47